Branch-SDK/Branch-SDK/BNCStrongMatchHelper.m

```diff
@@ -97,27 +95,28 @@ - (void)presentSafariVCWithBranchKey:(NSString *)branchKey {
 
     [urlString appendFormat:@"&sdk=ios%@", SDK_VERSION];
     
-    SFSafariViewController *safController = [[SFSafariViewController alloc] initWithURL:[NSURL URLWithString:urlString]];
-    safController.delegate = self;
-    
-    UIViewController *windowRootController = [[UIViewController alloc] init];
-    
-    self.secondWindow = [[UIWindow alloc] initWithFrame:CGRectZero];
-    self.secondWindow.rootViewController = windowRootController;
-    [self.secondWindow makeKeyAndVisible];
-    [self.secondWindow setAlpha:0];
-
-    [windowRootController presentViewController:safController animated:NO completion:NULL];
-}
-
-- (void)safariViewController:(SFSafariViewController *)controller didCompleteInitialLoad:(BOOL)didLoadSuccessfully {
-    [self.secondWindow.rootViewController dismissViewControllerAnimated:NO completion:NULL];
-
-    if (didLoadSuccessfully) {
-        [BNCPreferenceHelper preferenceHelper].lastStrongMatchDate = [NSDate date];
+    Class SFSafariViewControllerClass = NSClassFromString(@"SFSafariViewController");
+    if (SFSafariViewControllerClass) {
+        id safController = [[SFSafariViewControllerClass alloc] initWithURL:[NSURL URLWithString:urlString]];
+        
```

Comment:
I couldn't keep it unfortunately. I had to remove line 31 above because we're not importing it explicitly:  https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK/pull/177/files#diff-02041277e39d3db3a75c1036ed54f1d7L31

What we lose is the ability to know for sure that the request went through. So if the request times out, the cookie<>IDFA pair will not be established via this mechanism. Thankfully there are other backend mechanisms, and this will try again in 30 days. So marginal loss.